examples/with-typescript/pages/pagewithasync.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react'
+import 'isomorphic-fetch'
+interface pagesProps {
+  stars: number
+}
+interface pageState {
```

Comment:
interface names should use PascalCase just like classes. i prefer the `I` prefix convention too as it makes it obvious that this is a TS-only construct (and cannot be used in JS code): `IPageProps`, `IPageState`.

refer to any Blueprint component ([perhaps this one](https://github.com/palantir/blueprint/blob/master/packages/core/src/components/toast/toast.tsx)) for tried-and-true conventions.